danielssonn/ethTO
# chore: add nvmrc

## Purpose

1. Added an `.nvmrc` with the current LTS version, so that we're all using the same version of Node

## Testing Instructions

1. Install nvm, run `nvm use` and run `node -v` to confirm Node `16.16.0` is running (optional).
2. Review code

## Checklist
